python/ray/_private/workers/default_worker.py

```diff
@@ -229,6 +229,8 @@
     )
     configure_log_file(out_file, err_file)
 
+    utils.maybe_disable_object_store_coredumps()
```

Comment:
Could we add a test that the filter is set correctly for a task?

Reply:
For sure! I wanted to make sure this was the right place for this kind of change before fleshing it out. Any high-level feedback on this approach?

Reply:
Lgtm if it works.